pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -223,7 +238,7 @@ func (sync Synchronizer) syncOperationalState() error {
 	if len(subscribePaths) == 0 {
 		noPathErr := fmt.Errorf("target %#v has no paths to subscribe to", sync.ID)
 		log.Warning(noPathErr)
-		return noPathErr
+		return
```

Comment:
I don't think this is worth sending to the channel because we are doing no subscription to the device at this point in time. This means that nothing will be pulled from it in terms of STATE or OPERATIONAL values. Also I'm not sending up the error because I think there is really nothing we can do here. We are trying the get STATE and OPERATIONAL, if that fails we use the yang models (YET NOT DONE), if even that fails we have exhausted our options.